lib/http.js

```diff
@@ -123,7 +129,13 @@ module.exports = {
   validateHeaderName,
   validateHeaderValue,
   get,
-  request
+  request,
+  diagnosticsChannel: {
+    onClientRequestStart: onClientRequestStartChannel,
+    onClientResponseFinish: onClientResponseFinishChannel,
+    onHTTPRequestStart: onRequestStartChannel,
+    onHTTPResponseFinish: onResponseFinishChannel,
+  },
```

Comment:
The goal for diagnostic_channel is for these objects to not be exposed directly, but rather looked up via their logic name.

cc @nodejs/diagnostics

Reply:
> The goal for diagnostic_channel is for these objects to not be exposed directly, but rather looked up via their logic name.
> 
> cc @nodejs/diagnostics

Thanks for review ! Maybe i cloud put this logic name to document Because i don't see it in the document.

Reply:
That'd be great, yes!